x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/combine_renderers/index.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { Ecs } from '../../../../../../../common/ecs';
+import type { RowRenderer, RowRendererId } from '../../../../../../../common/types';
+
+export const combineRenderers = ({
```

Comment:
Awesome. I see a future where we would want this to be an array of renderers rather than just combining two. It would be an unnecessary optimization for now, but just a thought if you have time in the future for it